ui/src/notebooks/pipes/Query/index.ts

```diff
@@ -13,7 +13,8 @@ register({
     activeQuery: 0,
     queries: [
       {
-        text: '// Write Flux script here',
+        text:
+          '// Write Flux script here\n// use __PREVIOUS_RESULT__ to continue building from the previous cell',
```

Comment:
why this change?

Reply:
I added it in the query registration so it appears on all cells regardless of position. I figure showing it all the time doesn't impact usability